From d95d9499bbfb91757292427979afb1f1469143f4 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Fri, 21 Sep 2012 18:57:24 -0300
Subject: [RHEL6 qemu-kvm PATCH 03/23] usb-ehci: Drop unused sofv value

RH-Author: Hans de Goede <hdegoede@redhat.com>
Message-id: <1348253864-3050-3-git-send-email-hdegoede@redhat.com>
Patchwork-id: 42179
O-Subject: [RHEL-6.4 qemu-kvm PATCH 02/22] usb-ehci: Drop unused sofv value
Bugzilla: 805172
RH-Acked-by: Uri Lublin <uril@redhat.com>
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

The sofv value only ever gets a value assigned and is never used (read)
anywhere, so we can just drop it.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Upstream commit: 8e24283b2687e1d58d5f6a4872198c29e8a45d00
Conflicts: hw/usb-ehci.c
---
 hw/usb-ehci.c | 8 --------
 1 file changed, 8 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/usb-ehci.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/hw/usb-ehci.c b/hw/usb-ehci.c
index 875b526..35871b2 100644
--- a/hw/usb-ehci.c
+++ b/hw/usb-ehci.c
@@ -399,7 +399,6 @@ struct EHCIState {
     /*
      *  Internal states, shadow registers, etc
      */
-    uint32_t sofv;
     QEMUTimer *frame_timer;
     QEMUBH *async_bh;
     int attach_poll_counter;
@@ -1085,10 +1084,6 @@ static void ehci_mem_writel(void *ptr, target_phys_addr_t addr, uint32_t val)
         val &= USBINTR_MASK;
         break;
 
-    case FRINDEX:
-        s->sofv = val >> 3;
-        break;
-
     case CONFIGFLAG:
         val &= 0x1;
         if (val) {
@@ -2151,9 +2146,6 @@ static void ehci_frame_timer(void *opaque)
                 ehci_set_interrupt(ehci, USBSTS_FLR);
                 ehci->frindex = 0;
             }
-
-            ehci->sofv = (ehci->frindex - 1) >> 3;
-            ehci->sofv &= 0x000003ff;
         }
 
         if (frames - i > ehci->maxframes) {
-- 
1.7.11.4

